src/randoop/sequence/AbstractGenerator.java

```diff
@@ -113,40 +117,49 @@
   public static Sequence currSeq = null;
   
   /**
-   * The list of final sequences that are printed out as JUnit tests (i.e. Randoop's output). 
+   * The list of failure sequences to be saved as JUnit tests
+   */
+  public List<ExecutableSequence> outErrorSeqs = new ArrayList<>();
```

Comment:
Please use consistent terminology; here, the comment and the variable name use failure and error respectively.